5eShaped/translations/fr.json

```diff
@@ -543,6 +546,8 @@
   "REACTION":"Réaction",
   "REACTIONS":"Réactions",
   "RECHARGE":"Recharge",
+  "REDUCED_MAX":"Maximum réduit",
```

Comment:
Need to be smaller : "Max réduit" or "PV max réduits". I don't like how it sound, but don't have a better idea to keep it small.